lib/outputs/outputs.py

```diff
@@ -101,13 +105,16 @@ def report(check_findings, output_options, audit_info):
             file_descriptors.get(file_descriptor).close()
 
 
-def fill_file_descriptors(output_modes, audited_account, output_directory, csv_fields):
+def fill_file_descriptors(
+    output_modes, audited_account, output_directory, csv_fields, output_filename
+):
     file_descriptors = {}
+    # If no input custom output filename, set default:
+    if not output_filename:
+        output_filename = f"prowler-output-{audited_account}-{output_file_timestamp}"
```

Comment:
You are doing the same at lines 250 and 251, why don't you include this at a higher level so both functions always receive a non-empty `output_filename`. What do you think?

Reply:
Great idea! I put it at a high level now.